docs/bigquery-usage.rst

```diff
@@ -397,13 +386,17 @@ the job locally:
 .. doctest::
 
    >>> from gcloud import bigquery
+   >>> from gcloud.bigquery import SchemaField
    >>> client = bigquery.Client()
    >>> table = dataset.table(name='person_ages')
-   >>> job = table.load_from_storage(bucket_name='bucket-name',
-   ...                               object_name_glob='object-prefix*',
-   ...                               source_format='CSV',
-   ...                               skip_leading_rows=1,
-   ...                               write_disposition='truncate')
+   >>> table.schema = [
+   ...     SchemaField(name='full_name', type='string', mode='required'),
+   ...     SchemaField(name='age', type='int', mode='required)]
+   >>> job = client.load_table_from_storage(
+   ...          'from-storage', table, 'gs://bucket-name/object-prefix*')
```

Comment:
The `load_table_from_storage()` job factory actually takes one or more `source_uris`, which need to be in that form.